virtcontainers/hypervisor.go

```diff
@@ -668,7 +668,7 @@ type hypervisor interface {
 	hypervisorConfig() HypervisorConfig
 	getThreadIDs() (vcpuThreadIDs, error)
 	cleanup() error
-	pid() int
+	pid() []int
```

Comment:
Can you document the expected PID ordering here (is `pid[0]` "special" even if there are `>1` PIDs)?